src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$start-day: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$cellSize: 100px;
+$cellCollor: #eee;
```

Comment:
There seems to be a typo in the variable name `$cellCollor`. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variable names should be spelled correctly to avoid confusion. Consider renaming it to `$cellColor`.